components/brave_rewards/browser/BUILD.gn

```diff
@@ -71,12 +76,24 @@ source_set("browser") {
       "rewards_fetcher_service_observer.h",
     ]
 
+    if (!is_android) {
+      sources += [
+        "rewards_helper.cc",
+        "rewards_helper.h",
+      ]
+    }
+
     deps += [
       "//brave/vendor/bat-native-ledger",
       "//net",
       "//url",
-      # for one shot event
-      "//extensions/common",
     ]
+
+    if (!is_android) {
```

Comment:
can we move this above as an `else` https://github.com/brave/brave-core/pull/821/files#diff-0deb8db0aa3d14248a092d269a764b4dR45 to keep them together?

Reply:
but the above is not under `if (brave_rewards_enabled)`